### DISCUSSION TITLE
Storage source "container" supported?

### DISCUSSION THREAD

**Discussion by marcoshaw** [crc-org/crc#3468] [General]
I'm struggling getting virtualization to work, mainly it seems to be me failing to understand how to set up a PVC.

A Red Hat-er posted on LinkedIn that a VM can be created using a "container disk":

https://docs.openshift.com/container-platform/4.11/virt/virtual_machines/virt-create-vms.html#virt-storage-wizard-fields-web_virt-create-vms

![image](https://user-images.githubusercontent.com/2267513/209447563-86944a2b-eb77-4d06-955e-d446b0a2f6a3.png)

I don't see that option in in 4.11.13.  Might it be a future thing?

**Comment by praveenkumar** [comment]
You need to have kubevirt operator installed and then it should work, as per docs PVC will auto created so shouldn't be an issue. Make sure you put more resource to crc if you want to consume it for kubevirt.

**Reply by marcoshaw** [reply]
I'll test it out.  I guess I assumed that kubevirt = "OpenShift Virtualization".

**Reply by marcoshaw** [reply]
Some reading for me to do...  I've installed the Kubevirt operator.  I found another reference to needing to resize the CRC image before loading anything big into it.  It seems like there's a lot of pieces here and there unfortunately, and I still haven't gotten things to work.

**Reply by gbraad** [reply]
Kubervirt in a regular cluster would perform much better (and need less resources from a worker), as for our single node cluster everything runs in a single machine; backplane, worker and the VMs. This is therefore only sggested for testing purposes.
